src/database/schema.ts

```diff
@@ -18,6 +25,13 @@ export const users = pgTable('users', {
 export const usersRelations = relations(users, ({ one, many }) => ({
   auth: one(auth),
   session: many(session),
+  jobs: many(jobs),
+  ticketAssignee: many(tickets, {
+    relationName: 'ticket_assignee',
+  }),
+  ticket_creator: many(tickets, {
```

Comment:
just readability nitpick - you missmatch twoDifferent_cases

Reply:
in js world preferably 
js property name camelCase
sql proeprty snake_case

Reply:
In that case 
```js
export const tickets = pgTable('tickets', {
  ...createdAt,
  ...updatedAt,
  uuid: uuid('uuid')
    .default(sql`gen_random_uuid()`)
    .primaryKey(),
  title: varchar('title'),
  description: varchar('description'),
  user_uuid: uuid('user_uuid').references(() => users.uuid),
  creator_uuid: uuid('creator_uuid')
    .notNull()
    .references(() => users.uuid),
})```

all above js values should be camelCased?

Reply:
yeah so creatorUuid(creator_uuid)